src/inc-lib-variants.php

```diff
@@ -269,12 +270,58 @@ function lovd_fixHGVS ($sVariant, $sType = 'g')
                                 list($aPositions[$sIntronicFirst], $aPositions[$sIntronicLast]) =
                                     array($aPositions[$sIntronicLast], $aPositions[$sIntronicFirst]);
 
-                        } elseif ($sIntronicFirst == $sIntronicLast) {
-                            // INSERT SOLUTION
+                            } elseif ($sIntronicFirst == $sIntronicLast) {
+                                // INSERT SOLUTION
+                            }
                         }
                     }
                 }
             }
+
+        } else {
+            // In this case, a WTOOMUCHUNKNOWN warning was thrown.
+            // This means that question marks where given to the variant in
+            //  places where they do not bring any additional value. We
+            //  shall remove this redundancy by replacing the question marks
+            //  by empty strings, thus removing them from the variant.
+
+            if ($aPositions['C'] . $aPositions['D'] == '??') {
+                // e.g. c.1_(?_?)del
+                $aPositions['D'] = '';
+
+                // Fixme; have another look at the next three statements (range vs suffix).
+            } elseif ($aPositions['A'] . $aPositions['C'] == '??' && !$aPositions['B']) {
+                // e.g. c.?_(?_10)del
+                $aPositions['C'] = '';
+                $sBefore = $sBefore . '(';
+
+            } elseif ($aPositions['B'] . $aPositions['C'] == '??' && !$aPositions['D']) {
+                // e.g. c.(1_?)_?del
+                $aPositions['B'] = '';
+                $sAfter = ')' . $sAfter;
+
+            } elseif ($aPositions['B'] . $aPositions['C'] == '??' && $aPositions['A'] != '?' &&
+                      !in_array($aPositions['D'], array('', '?'))) {

```

Comment:
Whenever an if needs to be split in multiple lines, I find it more readable to put the `&&` or `||` on the new line instead of the end of the first line. On first glance, having the `&&` or `||` at the beginning of the new line makes it directly clear that line still belongs to the `if()` condition. Having it at the end of the previous line requires you to first look at whether or not the second line is the body of the `if()` or still the condition itself.